### PR TITLE
Fix risk watch zoom and tooltip issues

### PR DESCRIPTION
**Pull request by puranban**
### Addresses 
- [x] #2864 
- [x] https://github.com/toggle-corp/ifrc-go-meta/issues/230
- [x] https://github.com/toggle-corp/ifrc-go-meta/issues/235

### Changes

- Fix the GDACS zoom issue
- Fix the emergency surge personnel table tooltip
- Fix the home figures card tooltip

### CODE REVIEW

File: src/root/components/RiskImminentEventMap/GDACSEventMap/index.tsx [hunk 1]
```diff
@@ -233,33 +239,32 @@ function GDACSEventMap(props: Props) {
...
+      const pointBuffer = turfBuffer(point, 50, { units: 'kilometers' });
 
-        return viewPort;
-      }
+      const geojson = {
+        type: 'FeatureCollection',
+        features: [
+          pointBuffer,
+          ...(footprintPositionGeoJson?.features ?? []),
+        ].filter(isDefined),
```
Comment (tnagorra) [inline src/root/components/RiskImminentEventMap/GDACSEventMap/index.tsx:258]: Any reason we need to filter?

File: src/root/views/Country/RiskWatch/ImminentEventsGDACS/index.tsx [hunk 2]
```diff
@@ -102,6 +104,13 @@ function ImminentEventsGDACS(props: Props) {
     url: `risk://api/v1/gdacs/${eventId}/exposure`,
   });
 
+  const countryBounds = React.useMemo(
+    () => {
+      let bbox = turfBbox(country?.bbox ?? []);
```
Comment (tnagorra) [inline src/root/views/Country/RiskWatch/ImminentEventsGDACS/index.tsx:109]: Can we even pass empty array on turfBbox function?
Why are we passing bbox on turfBbox?

File: src/root/views/RegionalRiskWatch/ImminentEvents/ImminentEventsGDACS/index.tsx [hunk 3]
```diff
@@ -96,6 +98,13 @@ function ImminentEventsGDACS(props: Props) {
     url: `risk://api/v1/gdacs/${eventId}/exposure`,
   });
 
+  const regionBounds = React.useMemo(
+    () => {
+      let bbox = turfBbox(region?.bbox ?? []);
```
Comment (tnagorra) [inline src/root/views/RegionalRiskWatch/ImminentEvents/ImminentEventsGDACS/index.tsx:103]: Can we even pass empty array on turfBbox function?
Why are we passing bbox on turfBbox?

**Review by tnagorra — CHANGES_REQUESTED** [review]

File: src/root/views/Country/RiskWatch/ImminentEventsGDACS/index.tsx (hunk 2 above)
Comment (frozenhelium) [reply src/root/views/Country/RiskWatch/ImminentEventsGDACS/index.tsx:109]: We don't get the bounds format that is supported by mapbox, we transform it to correct format using turf

**Review by frozenhelium — APPROVED** [review]

**Pull request merged**